doc/source/edits.rst

```diff
@@ -849,6 +849,28 @@ Examples:
 
     order GHC.Base.Functor__arrow GHC.Base.Applicative__arrow_op_ztzg__ GHC.Base.Applicative__arrow GHC.Base.Monad__arrow_return_ GHC.Base.Monad__arrow GHC.Base.Alternative__arrow GHC.Base.MonadPlus__arrow
 
+
+``promote`` -- promote a definition from the term level to the type level
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index::
+   single: promote, edit
+
+Format:
+  | **promote** *qualified_name* ...
+
+Effect:
+  ``hs-to-coq`` divides Haskell definitions into two kinds: type-level and term-level.
+  The ``promote`` edit "prmotes" a term-level definition to the type level.
```

Comment:
I'm thinking if this description here would be a little bit misleading. People may incorrectly interpret this edit as something that works with the data type promotion extension of Haskell (i.e. `DataKinds`).

I think one thing we have not made clear in our previous documentation is that, in a translated Coq file, type-level definitions would appear before term-level definitions. This edit (if I understand correctly) does not try to change anything about the original definition. Instead, it moves the definition to the region dedicated to type-level definitions. Maybe the description for this edit can try to explain a bit of this? What do you think?

Reply:
Sure, I have modified the description for this edit to clarify this. Let me know if this is an improvement.